py/client/pydeephaven/_table_ops.py

```diff
@@ -2,41 +2,67 @@
 # Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
 #
 from abc import ABC, abstractmethod
+from enum import Enum
 from typing import List, Any
 
 import pyarrow as pa
 
 from pydeephaven._arrow import map_arrow_type
 from pydeephaven.agg import Aggregation
-from pydeephaven.constants import SortDirection, MatchRule
 from pydeephaven.proto import table_pb2, table_pb2_grpc
 from pydeephaven.updateby import UpdateByOperation
 
 
+class MatchRule(Enum):
+    """An enum defining the match rules for the as-of and reverse-as-of joins."""
+
+    """"""
```

Comment:
For all of the enum values, I thought we were adding descriptions everywhere for enum values.